app/src/main/res/values-de/strings.xml

```diff
@@ -291,4 +291,5 @@ work. If not, see <https://creativecommons.org/publicdomain/zero/1.0/>.
     <string name="share_into">Teilen nach</string>
     <string name="favourite">Favorisieren</string>
     <string name="favourites">Favoriten</string>
+    <string name="text_words">Text Wörter</string>
```

Comment:
Translation is done on Crowdin.

This will get automatically deleted by the integration. Please revert and do the translation on Crowdin after merge :).

BTW, happy to see another german translator beside me.